test/app/App/Notify/AddChain/index.test.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import Restore from 'react-restore'
+import { render } from '@testing-library/react'
+
+import store from '../../../../../main/store'
+import AddChainComponent from '../../../../../dash/App/Notify/AddChain'
+
+jest.mock('../../../../../main/store/persist')
+
+const AddChain = Restore.connect(AddChainComponent, store)
+
+const requestProps = (chain = {}) => {
+  return {
+    editMode: false,
+    req: { chain }
+  }
+}
+
+beforeAll(() => {
+  jest.useFakeTimers()
+
+  store.removeNetwork({ type: 'ethereum', id: 137 })
+  store.addNetwork({
+    id: 1,
+    type: 'ethereum',
+    name: 'Mainnet',
+    explorer: 'https://etherscan.io',
+    symbol: 'ETH',
+    on: true,
+    connection: {
+      primary: { connected: true }
+    }
+  })
+})
+
+afterAll(() => {
+  jest.useRealTimers()
+})
+
+describe('adding a new chain', () => {
+  it('renders with defaults', () => {
+    const { getByLabelText, getByRole } = render(<AddChain {...requestProps()} />)
+
+    const chainNameInput = getByLabelText('Chain Name')
+    expect(chainNameInput.value).toEqual('Chain Name')
+
+    const chainIdInput = getByLabelText('Chain ID')
+    expect(chainIdInput.value).toEqual('Chain ID')
+
+    const blockExplorerInput = getByLabelText('Block Explorer')
+    expect(blockExplorerInput.value).toEqual('Block Explorer')
+
+    const primaryRpcInput = getByLabelText('Primary RPC')
+    expect(primaryRpcInput.value).toEqual('Primary Endpoint')
+
+    const secondaryRpcInput = getByLabelText('Secondary RPC')
+    expect(secondaryRpcInput.value).toEqual('Secondary Endpoint')
+
+    const otherLayerButton = getByRole('radio', { name: 'Other' })
+    expect(otherLayerButton.getAttribute('aria-checked')).toBe('true')
```

Comment:
I'd invert this to look for the checked button in the first instance, something like
```
const checkedLayerButton = getByRole('radio', { checked: true })
expect(checkedLayerButton.textContent).toBe('Other')
```

Reply:
:+1: